gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
@cindyli, I have fixed the duplicate test and the broken path.  Both of those are great catches, thanks as always for your incredible eye for detail.

I also updated the README with a warning about the 404 errors when hosting the tests yourself.  Please review, I hope that will be good enough for now.